README.MD

```diff
@@ -264,11 +248,10 @@ const signature = workerAccount.signEIP712TypedData(stakeRequestTypedData);
 ### Facilitator calls acceptStakeRequest
 
 ```js
-const Mosaic = require('@openstfoundation/mosaic.js');
-const hashLockInstance = Mosaic.Helpers.StakeHelper.createSecretHashLock();
-const gatewayContractInstance = Mosaic.Contracts.getEIP20Gateway(web3Provider, gatewayAddress);
+const hashLockInstance = Mosaic.Utils.createSecretHashLock();
+const gatewayContractInstance = new Mosaic.ContractInteract.EIP20Gateway(web3Provider, gatewayAddress);
 let bountyAmountInWei;
-gatewayContractInstance.methods.bounty().call().then(function(result){
+gatewayContractInstance.getBounty().then(function(result){
```

Comment:
This in conjunction with lines 255 and 264 leads to a racing condition, if I see that correctly. Please fix.

Reply:
Sorry I didn't get the comment. Please elaborate?

Reply:
I think I understood it. You are saying Promise will not resolve sequentially as expected.
Please confirm once?

Reply:
Added a comment here:
https://github.com/OpenSTFoundation/brandedtoken.js/pull/98/files#r257645160

Reply:
OK. It will be handled in my PR.